examples/TensorFlow_on_California_housing_prices_dataset.ipynb

```diff
@@ -0,0 +1,1309 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "id": "36bcbf97",
+      "metadata": {
+        "id": "36bcbf97"
+      },
+      "source": [
+        "# New Relic ML Performance Monitoring - Bring Your Own Data\n",
+        "\n",
+        "[ml-performance-monitoring](https://github.com/newrelic-experimental/ml-performance-monitoring) provides a Python library for sending machine learning models' inference data and performance metrics into New Relic.\n",
+        "<br>\n",
+        "By using this package, you can easily and quickly monitor your model, directly from a Jupyter notebook or a cloud service.\n",
+        "<br>\n",
+        "The package is ML framework agnostic and can be quickly integrated. It is based on the newrelic-telemetry-sdk-python library.\n",
+        "<br>\n",
+        "It is based on the [newrelic-telemetry-sdk-python](https://github.com/newrelic/newrelic-telemetry-sdk-python) library.\n",
+        "\n",
+        "\n",
+        "This notebook provides an example of sending inference data and metrics of an XGBoost model\n",
+        "\n",
+        "<U>Note</U>- this notebook uses the libraries:\n",
+        "* numpy\n",
+        "* pandas\n",
+        "* sklearn\n",
+        "* tensorflow"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "77eda1c8",
+      "metadata": {
+        "id": "77eda1c8"
+      },
+      "source": [
+        "### 0. Install libraries"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 3,
+      "id": "e984c34f",
+      "metadata": {
+        "id": "e984c34f",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "outputId": "62febb56-0560-480b-8b48-1730ca48270d"
+      },
+      "outputs": [
```

Comment:
Is there any reason to keep the output from your tests in the file? Or should the outputs be cleared before merging? I did not check how many there were, but there are a number of output streams included in the file.

Reply:
I saw that the XGBoost example includes some output so I thought I should include my output as well. But I have now cleared the output, pushed a new commit and squashed the commits. There's no output now.

Reply:
Thank you! Yes, it is a bit inconsistent, but we'll get that sorted out.

Reply:
Thanks!